src/components/PrimaryPanes/index.js

```diff
@@ -22,6 +22,11 @@ import SourcesTree from "./SourcesTree";
 
 import type { SourcesMap } from "../../reducers/types";
 
+import Tab from "react-aria-components/src/tab";
+import Tabs from "react-aria-components/src/tabs";
+import TabList from "react-aria-components/src/tab-list";
+import TabPanels from "react-aria-components/src/tab-panels";
```

Comment:
The 4 separate imports isn't ideal, and is WIP for now. I'm intending to update react-aria-components so that we can do:

```jsx
import { Tab, Tabs, TabList, TabPanels } from 'react-aria-components/src/tabs'
```

Reply:
CC @yzen, @darkwing, this is pretty cool. I don't know if we can use it yet because we need React 16.3, but wanted to share

Reply:
This is really nice, just looking through the repo it seems to be doing the right stuff.